src/components/PeopleTable.tsx

```diff
@@ -1,645 +1,80 @@
-          </td>
-          <td>m</td>
-          <td>1832</td>
-          <td>1905</td>
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/maria-van-brussel-1801"
-            >
-              Maria van Brussel
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carel-haverbeke-1796">Carel Haverbeke</a>
-          </td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a className="has-text-danger" href="#/people/maria-sturm-1835">
-              Maria Sturm
-            </a>
-          </td>
-          <td>f</td>
-          <td>1835</td>
-          <td>1917</td>
-          <td>Seraphina Spelier</td>
-          <td>Charles Sturm</td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/emma-de-milliano-1876"
-            >
-              Emma de Milliano
-            </a>
-          </td>
-          <td>f</td>
-          <td>1876</td>
-          <td>1956</td>
-          <td>Sophia van Damme</td>
-          <td>Petrus de Milliano</td>
-        </tr>
-
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/emile-haverbeke-1877">Emile Haverbeke</a>
-          </td>
-          <td>m</td>
-          <td>1877</td>
-          <td>1968</td>
-          <td>
-            <a className="has-text-danger" href="#/people/maria-sturm-1835">
-              Maria Sturm
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carolus-haverbeke-1832">Carolus Haverbeke</a>
-          </td>
-        </tr>
-      </tbody>
-    </table>
+  return (
+    <div className="column">
+      <div className="box table-container">
+        {people.length ? (
+          <table
+            data-cy="peopleTable"
+            className="table is-striped is-hoverable is-narrow is-fullwidth"
+          >
+            <thead>
+              <tr>
+                {Object.entries(SortFilter).map(([key, value]) => (
+                  <th key={value}>
+                    <span className="is-flex is-flex-wrap-nowrap">
+                      {key}
+                      <Link
+                        to={{
+                          search: getSearchWith(searchParams, {
+                            sort:
+                              !sort || !order || sort !== value ? value : null,
+                            order: sort === value && !order ? Order.desc : null,
+                          }),
+                        }}
+                      >
+                        <span className="icon">
+                          <i
+                            className={classNames('fas', {
+                              'fa-sort': sort !== value,
+                              'fa-sort-up': !order && sort && sort === value,
+                              'fa-sort-desc': order === Order.desc,
+                            })}
```

Comment:
The logic for determining the class name of the icon is a bit complex and is repeated for each header cell. Consider extracting this logic into a separate helper function or a custom hook to avoid repetition and improve readability.